docs/node/run-your-node/validator-node/README.md

```diff
@@ -1,29 +1,48 @@
+This guide will walk you through the process of setting up your **validator node** 
+for the Oasis Network on either the Mainnet or the Testnet. It is designed for 
+individuals who have a basic understanding of command line tools. 
 
-:::
+## Prerequisites
 
-This guide will cover setting up your validator node for the Oasis Network. This guide assumes some basic knowledge on the use of command line tools.
+Before proceeding with this guide, ensure that you have completed the steps 
+outlined in the [Prerequisites] section so that:
+* your system meets the [hardware requirements],
+* you have the [Oasis Node binary] installed on your systems,
+* you understand what are [Stake requirements] to become a validator on the
+ Oasis Network.
 
-## Prerequisites
+[Prerequisites]: ../prerequisites/
+[hardware requirements]: ../prerequisites/hardware-recommendations.md
+[Oasis Node binary]: ../prerequisites/oasis-node.md
+[Stake requirements]: ../prerequisites/stake-requirements.md
 
-Before following this guide, make sure you've followed the [Prerequisites](../prerequisites/) section and have the Oasis Node binary installed on your systems.
 
-### Stake requirements
+## Physical Machines
 
-To become a validator on the Oasis Network, you need to have enough tokens staked in your escrow account. For more details, see the [Stake requirements](../prerequisites/stake-requirements.md).
+You will be using two separate physical machines for deployment: your local
+system, referred to as `localhost`, and a remote `server` which will function
+ as an Oasis node. The reason for this is to ensure protection of the keys used
+ to setup your node.
 
-### Systems
+**Localhost:** On the localhost, you will [create your entity] which is
+essential for deploying nodes on the network. To ensure the security of these
+private keys, we strongly recommend to isolate the localhost from any network or
+internet connectivity. Once the keys have been generated, it is highly
+recommended to use Hardware Security Modules (HSM) for secure storage
+such as the [Ledger wallet].
 
-This guide assumes that you have two different physical machines that you will use for deployment. These machines are the following:
+[Ledger wallet]:
+  ../../../general/manage-tokens/holding-rose-tokens/ledger-wallet
 
-* Your local system, henceforth called the `localhost`.
-* A remote system to run as an Oasis node, henceforth called the `server`.
+**Server:** After the entity has been created, proceed to [set up the server in
+ order to start the Oasis node]. The server needs to meet the hardware 
+ requirements and have an acces to the internet.
 
-The reason for this is to ensure protection of the keys used to setup your node. Use of HSMs to store keys is highly recommended.
+[create your entity]: ./#creating-your-entity
```

Comment:
I think `./` is redundant in this case. See https://docusaurus.io/docs/markdown-features/toc#heading-ids
```suggestion
[create your entity]: #creating-your-entity
```